docs/getting-started-guides/azure.md

```diff
@@ -0,0 +1,104 @@
+---
+---
+
+* TOC
+{:toc}
+
+## Prerequisites
+
+1. An Azure Subscription. If you don't already have one, you may create one on [azure.microsoft.com](https://azure.microsoft.com).
+2. A Microsoft account with access to your own Subscription, or an organizational account with Owner access to the Subscription.
```

Comment:
I wonder if mentioning MSA requirement is going to cause extra confusion. People who have an Azure subscription probably (>99%) already have a MS account and they are the owners of the subscription?